service-api/src/main/java/greencity/service/HabitAssignService.java

```diff
@@ -197,15 +197,15 @@ List<UserShoppingAndCustomShoppingListsDto> getListOfUserAndCustomShoppingListsW
     HabitAssignManagementDto updateStatusByHabitIdAndUserId(Long habitId, Long userId, HabitAssignStatDto dto);
 
     /**
-     * Method to enroll {@code Habit}.
+     * Method to enroll {@code HabitAssign} by habitAssignId.
      *
-     * @param habitId  {@code Habit} id to enroll.
-     * @param userId   {@code User} id.
-     * @param dateTime {@link LocalDate} dateTime we want enroll.
-     * @param language {@link String} of language code value.
+     * @param habitAssignId {@code HabitAssign} id to enroll.
+     * @param userId        {@code User} id.
+     * @param dateTime      {@link LocalDate} dateTime we want enroll.
```

Comment:
`LocalDate` does not store or represent a time or time-zone https://docs.oracle.com/javase/8/docs/api/java/time/LocalDate.html